new_release_notes.sh

```diff
@@ -90,7 +93,7 @@ then
   echo;echo "Here is how you can generate a file containing the list of github issues :"
```

Comment:
Can we bring these steps inside the script as well and automate it so the user will have one less step to be worried about?

Reply:
Will try to do in another pr.